R/df-helpers.R

```diff
@@ -38,12 +38,18 @@ get_metadataType_indices <- function(data, meta_types) {
 #' @param species The species needed to specify the correct biospecimen
 #' or individual templates (default `NA`).
 #' @param assay The assay needed to specify the correct assay template.
+#' @param biospecimen_type The type of biospecimen template needed
```

Comment:
Are accepted types "in_vitro" or "other"? What will happen if another value is passed?

Reply:
This accepts any values from the biospecimen template section of the config. The `biospecimen_type`s that will appear in the UI options will be pulled from the list of named biospecimen templates in the config. This adds extra flexibility for more types of biospecimen templates in the future, and gives you control over how the `biospecimen_type` appears in the app.

Reply:
fabulous.